united-states-international-trade-commission.csl

```diff
@@ -371,7 +422,17 @@
         </names>
         <text variable="title" form="short" text-case="title" quotes="true"/>
       </else-if>
-      <else-if type="webpage" match="any"/>
+      <else-if type="post-weblog" match="all" variable="author">
+        <choose>
+          <if match="any" variable="title-short">
+            <text variable="title-short" quotes="true"/>
```

Comment:
the difference with respect to quotation marks for short & full title here is on purpose?